tests/integration/cartography/intel/aws/ec2/test_ec2_instances.py

```diff
@@ -90,3 +93,59 @@ def test_ec2_reservations_to_instances(neo4j_session, *args):
         for n in nodes
     }
     assert actual_nodes == expected_nodes
+
+
+def test_ec2_iaminstanceprofiles(neo4j_session, *args):
+    """
+    Ensure that iaminstanceprofiles are connected to their expected iam roles
```

Comment:
Nit: "Ensure that EC2Instances are attached to the IAM Roles that they can assume due to their IAM instance profiles".